ingestors/bookmarks.go

```diff
@@ -0,0 +1,34 @@
+package ingestors
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/librariesio/depper/redis"
+)
+
+// Use to set a date/string bookmark for an ingestor
+func setBookmark(namer Namer, bookmark interface{}) (interface{}, error) {
```

Comment:
These accept and return interfaces because I suspect it'll commonly be a date, but for NPM is a sequence number string. These lets us use whatever.